rai/client.go

```diff
@@ -601,6 +601,28 @@ func (c *Client) ListEngines(filters ...interface{}) ([]Engine, error) {
 	return result.Engines, nil
 }
 
+func (c *Client) ResumeEngine(engineName string) error {
+	var result interface{}
```

Comment:
so there's no response body?

Reply:
there is not a response, the body is empty.